label_studio_tools/core/label_config.py

```diff
@@ -86,13 +106,15 @@ def parse_config(config_string):
             }
         if tag.tag not in _LABEL_TAGS:
             continue
+
         parent_name = _get_parent_output_tag_name(tag, outputs)
         if parent_name is not None:
             actual_value = tag.attrib.get('alias') or tag.attrib.get('value')
             if not actual_value:
                 logger.debug(
                     'Inspecting tag {tag_name}... found no "value" or "alias" attributes.'.format(
-                        tag_name=etree.tostring(tag, encoding='unicode').strip()[:50]
+                        tag_name=etree.tostring(
```

Comment:
let's keep this line untouched?

Reply:
Ahhh, the auto linter struck again! Will put back